src/usr/local/www/firewall_rules.php

```diff
@@ -1226,7 +1226,7 @@ function($type) {
 
 	// Provide a warning message if the user tries to change page before saving
 	$(window).bind('beforeunload', function(){
-		if ((!saving && dirty) || newSeperator) {
+		if ((!saving && dirty) || newSeparator) {
```

Comment:
Bug. This variable name contains a typo - it was missed in a previous commit which changed the original variable name. See previous comments for more details.